README.md

```diff
@@ -39,43 +43,83 @@ public
 
 ## Agent
 
-### Getting and running the agent
+### Install
 
 Agent binaries can be found under wiresteward releases:
 https://github.com/utilitywarehouse/wiresteward/releases
 
-to install on linux simply:
+To install on linux simply:
```

Comment:
Since we are changing README, can we remove all the "simply"? Its redundant at best and at worst patronising.

Reply:
done